### PR TITLE
Add short intro and to dictionary to zha.markdown

### PR DESCRIPTION
**Pull request by Hedda**
## Proposed change

Added a short ZHA introduction with Zigbee diction and ZHA limitations as a summary to zha.markdown meant for new end users.

I think this will help explain the most essential questions that many potential ZHA users (and also users new to Zigbee) commonly have and ask before they buy an adapter and install the ZHA integration or first asked directly after they have installed ZHA. 

The purpose of this short introduction is to help new users quickly get up to speed on the hardware requirements for ZHA as well as mention the minimum essential Zigbee terminology that is really needed to get easily up and running with the ZHA integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zha.markdown [hunk 1]
```diff
@@ -64,6 +64,24 @@ There is currently support for the following device types within Home Assistant:
 
 There is also support for grouping of lights, switches, and fans (i.e. support for commanding device groups as entities). At least two entities must be added to a group before the group entity is created. As well as support for binding/unbinding (i.e. bind a remote to a lightbulb or group).
 
+## Introduction and dictionary (terminology vocabulary)
+
+ZHA is a native “Zigbee host application” for Home Assistant that via a Zigbee Coordinator adapter will setup and allow control of a single “Zigbee network” (using mesh topology). Once installed ZHA you will be able to directly join/pair most types of “Zigbee Router Device” and “Zigbee End Device” products to that Zigbee network, regardless of product manufacturer and brand.
```
Comment (frenck) [inline source/_integrations/zha.markdown]: A user coming and reading this for the first time, will have no clue what all these things even mean.

File: source/_integrations/zha.markdown [hunk 2]
```diff
@@ -64,6 +64,24 @@ There is currently support for the following device types within Home Assistant:
...
+
+Zigbee Router Devices will interconnect with other devices to distribute and repeat transmission messages within the Zigbee network mesh, (they will therefor always need be available so are normally mains-powered).  Zigbee End Devices are usually but not always low-powered products such as battery-operated sensors, these will “sleep” most of the time to save power and only “wake” to report in when changes occur, (example sensor state change like temperature or open and close).
```
Comment (frenck) [inline source/_integrations/zha.markdown]: I don't see how this is relevant in the introduction for the integration. Its also filled with technical terms that most users won't have any idea what it means.
Let simplify it.

File: source/_integrations/zha.markdown [hunk 3]
```diff
@@ -64,6 +64,24 @@ There is currently support for the following device types within Home Assistant:
...
 
+## Introduction and dictionary (terminology vocabulary)
+
+ZHA is a native “Zigbee host application” for Home Assistant that via a Zigbee Coordinator adapter will setup and allow control of a single “Zigbee network” (using mesh topology). Once installed ZHA you will be able to directly join/pair most types of “Zigbee Router Device” and “Zigbee End Device” products to that Zigbee network, regardless of product manufacturer and brand.
+
+Zigbee Router Devices will interconnect with other devices to distribute and repeat transmission messages within the Zigbee network mesh, (they will therefor always need be available so are normally mains-powered).  Zigbee End Devices are usually but not always low-powered products such as battery-operated sensors, these will “sleep” most of the time to save power and only “wake” to report in when changes occur, (example sensor state change like temperature or open and close). 
+
+In most common Zigbee setups you will always need to first setup many Zigbee Router Devices to extend range and coverage of the Zigbee network, otherwise signals and/or messages to far way devices might be lost.
```
Comment (frenck) [inline source/_integrations/zha.markdown]: What does this have to do with an introduction?

File: source/_integrations/zha.markdown [hunk 4]
```diff
@@ -64,6 +64,24 @@ There is currently support for the following device types within Home Assistant:
...
+
+#### ZHA and Zigbee limitations
```
Comment (frenck) [inline source/_integrations/zha.markdown]: Let move limitations to a lower part of the documentation.

File: source/_integrations/zha.markdown [hunk 5]
```diff
@@ -64,6 +64,24 @@ There is currently support for the following device types within Home Assistant:
...
+  - ZHA only support connecting one Zigbee Coordinator adapter / one Zigbee network.
+    - Zigbee devices can only be joined/paired to one Zigbee Coordinator / connected to one Zigbee network.
+    - Zigbee 3.x specification is limited to one Zigbee Coordinator per Zigbee network.
+  - Newer ZHA compatible Zigbee Coordinator adapters support Zigbee 3.x (dependecy on “zigpy” library)
+    - Recommendation is to buy a newer Zigbee Coordinator and flash latest firmware for best compatibility.
+    - ZHA’s dependency library “zigpy” does not yet support ZGP (Zigbee Green Power)
```
Comment (frenck) [inline source/_integrations/zha.markdown]: The end user should not have to care about the library used. Just state Zigbee Green Power isn't supported yet.

File: source/_integrations/zha.markdown [hunk 6]
```diff
@@ -64,6 +64,24 @@ There is currently support for the following device types within Home Assistant:
...
+
+Zigbee Router Devices will interconnect with other devices to distribute and repeat transmission messages within the Zigbee network mesh, (they will therefor always need be available so are normally mains-powered).  Zigbee End Devices are usually but not always low-powered products such as battery-operated sensors, these will “sleep” most of the time to save power and only “wake” to report in when changes occur, (example sensor state change like temperature or open and close). 
+
+In most common Zigbee setups you will always need to first setup many Zigbee Router Devices to extend range and coverage of the Zigbee network, otherwise signals and/or messages to far way devices might be lost.
+
+#### ZHA and Zigbee limitations
+  - ZHA only support connecting one Zigbee Coordinator adapter / one Zigbee network.
+    - Zigbee devices can only be joined/paired to one Zigbee Coordinator / connected to one Zigbee network.
```
Comment (frenck) [inline source/_integrations/zha.markdown]: This is not an integration limitation.

File: source/_integrations/zha.markdown [hunk 7]
```diff
@@ -64,6 +64,24 @@ There is currently support for the following device types within Home Assistant:
...
+    - Zigbee 3.x specification is limited to one Zigbee Coordinator per Zigbee network.
+  - Newer ZHA compatible Zigbee Coordinator adapters support Zigbee 3.x (dependecy on “zigpy” library)
+    - Recommendation is to buy a newer Zigbee Coordinator and flash latest firmware for best compatibility.
+    - ZHA’s dependency library “zigpy” does not yet support ZGP (Zigbee Green Power)
+  - Devices not following specifications might first need a custom ZHA Device Handler ("quirk") to work.
+    - ZHA support most devices out-of-the-box + already have "quirks" for many deviating from set standards.
```
Comment (frenck) [inline source/_integrations/zha.markdown]: This sentence seems like the end-user need to do something, but the call to action is missing; leaving the end user clueless.

**Review by frenck — CHANGES_REQUESTED** [review]
Some initial comments.

Don't get me wrong @Hedda, but you already have PRs open for this page, with similar readability comments. We need to target end-users.

I would very much appreciate if you'd finish up those comments first before opening new PRs.

Thanks 👍 

../Frenck

**Comment by Hedda** [comment]
> Some initial comments.
> 
> Don't get me wrong @Hedda, but you already have PRs open for this page, with similar readability comments. We need to target end-users.
> 
> I would very much appreciate if you'd finish up those comments first before opening new PRs.

@frenck If you are referring to https://github.com/home-assistant/home-assistant.io/pull/18864 then I already rewrote that PR based on feedback and submitted update (see -> https://github.com/home-assistant/home-assistant.io/commit/970295a277e8f01d3ee39eeeaacf453625b988d3).

https://github.com/home-assistant/home-assistant.io/pull/18864 was initially submitted on 9 Aug 2021 and took 6-month before first review so think justity new PR for a separate section.

**Comment by frenck** [comment]
> If you are referring to https://github.com/home-assistant/home-assistant.io/pull/18864 then I already rewrote that all based on feedback and submitted update

Yes I meant that one.

File: source/_integrations/zha.markdown (hunk 6 above)
Comment (Hedda) [reply source/_integrations/zha.markdown]: They actually are. ZHA could add support for an additional Zigbee Coordinator adapter and allow setting up a separate network on that, (I read that is something that Zigbee2MQTT is planning to add support for). 

Newer Zigbee Coordinator stacks/firmware from Silicon Labs and does also support "multi-PAN" (multi-network) which allows having more than one Zigbee network on a single  Zigbee Coordinator adapter. 

A real-world scenario example for those is when for example a "production" Zigbee network for stable devices and a "test" network for unstable and troublesome devices. instance. Another example is having older non-Zigbee 3.0 devices on one Zigbee Coordinator and newer Zigbee 3.0 devices on a different Zigbee Coordinator as they have different security, as that way newer Zigbee 3.0 devices do not need to run lower security to be compatible with older devices.

An alternative could be for Home Assistant to allow the installation of multiple instances of the ZHA integration as that way users could connect several Zigbee Coordinator adapters (with seperate networks) to same instance of HA.

PS: You can already use multiple Zigbee Coordinators with Zigbee2MQTT by adding remote Zigbee2MQTT instances.

File: source/_integrations/zha.markdown (hunk 7 above)
Comment (Hedda) [reply source/_integrations/zha.markdown]: Yes just missing link redirecting to the "zha#zha-exception-and-deviation-handling" section?

File: source/_integrations/zha.markdown (hunk 5 above)
Comment (Hedda) [reply source/_integrations/zha.markdown]: ok

File: source/_integrations/zha.markdown (hunk 4 above)
Comment (Hedda) [reply source/_integrations/zha.markdown]: ok

File: source/_integrations/zha.markdown (hunk 6 above)
Comment (frenck) [reply source/_integrations/zha.markdown]: > Zigbee devices can only be joined/paired to one Zigbee Coordinator / connected to one Zigbee network.

This line has nothing to do with the integration. Nothing in this line of documentation indicates that or is related to the integration. You last comment is probably thinking about something else. I'll ignore.

File: source/_integrations/zha.markdown (hunk 7 above)
Comment (frenck) [reply source/_integrations/zha.markdown]: I think it can be removed.

File: source/_integrations/zha.markdown (hunk 3 above)
Comment (Hedda) [reply source/_integrations/zha.markdown]: Maybe more an introduction to Zigbee than to ZHA specifically but I think most/many people do not know this before installing and I think it certainly helps understand that ZHA/Zigbee needs to communicate in a "mesh".

As a backstory, I have seen many new users who right away just ordered a Zigbee Coordinator and loads of battery-operated sensors before installing ZHA then asking in the community why can not connect to sensors far away.

Comment (frenck) [reply source/_integrations/zha.markdown]: > Maybe more an introduction to Zigbee than to ZHA specifically 

This page is not about Zigbee, its about ZHA.

**Comment by Hedda** [comment]
@frenck by the way, watched this video https://www.youtube.com/watch?v=MfJ5OyNwuHc&ab_channel=FranckNijhof by yourself a year back or so and remember thinking that even you too had a hard time getting started with ZHA for the very first time. 

Same reason why I like to help improve docs to assist both beginners as well as more advanced users coming with experiences from other Zigbee implementations or alternative protocols new to ZHA get a smooth and seamless start with a stable network.

Personally, I had a lot of previous experience with Z-Wave and my first impression was that Zigbee is an unstable mess with horrible interoperability, but I think that my first impression would have been way different if the ZHA docs contained some Zigbee basics so new would not want to go elseware to learn essentials on how Zigbee works,  essentials which every Zigbee user should really know to not have a unfairly bad experience getting started.

**Comment by frenck** [comment]
> by yourself a year back or so and remember thinking that even you too had a hard time getting started with ZHA for the very first time.

Its not a good reference, things changed a lot. Those were UI issues which have been actively addressed and resolved. It also has 0 to do with what you are writing here, nor does it justify anything.

> Same reason why I like to help improve docs to assist both beginners

Lets make stuff less technical and readable for that reason instead of discussing it. 


Thanks 👍 

../Frenck

**Comment by Hedda** [comment]
OK. Please see my rewrite -> https://github.com/home-assistant/home-assistant.io/pull/22260/commits/3adb003d957d0a562a18037a77d675f866c00d64

## Introduction

ZHA integration uses one Zigbee Coordinator adapter to set up and control a Zigbee network, (please see the section below for compatible hardware). Once installed ZHA you will be able to directly join/pair Zigbee Router Device and Zigbee End Device products to that Zigbee network, regardless of the manufacturer and brand of that product.

Supported devices are Zigbee Router Devices, which are normally mains-powered that act repeaters within the Zigbee network mesh to extend its range and coherence. And Zigbee End Devices, which will not act as repeaters and are usually but not always battery-operated sensors. ZHA does not yet support "Zigbee Green Power" devices, (such as for example "Friends of Hue" battery-less switches). Any devices that are or have previously been connected to another Zigbee implementation will need to first be reset to factory default before they can be paired/joined to ZHA.

Before installing the integration you will need to buy and connect a Zigbee Coordinator adapter to the computer running the Home Assistant installation. The general recommendation is to buy a newer model of Zigbee Coordinator adapter hardware and to flash the latest firmware to it, the reason for that is that it will usually offer better interoperability with all functions of most Zigbee devices on the market. Note that ZHA only supports connecting one Zigbee Coordinator adapter with a single Zigbee network.
```

**Comment by Hedda** [comment]
Not stale, waiting for review.

**Comment by Hedda** [comment]
Updated end user diction introduction while waiting for a review of the rewrite.

**Comment by Hedda** [comment]
Not stale (beating the github-actions stale message to the punch).

**Comment by Hedda** [comment]
Not stale, the rewritten text has not been reviewed.

**Comment by Hedda** [comment]
Closing in favour of https://github.com/home-assistant/home-assistant.io/pull/24494

**Comment by frenck** [comment]
I'm not sure why you didn't update this PR instead.
